app/models/actor.rb

```diff
@@ -0,0 +1,6 @@
+class Actor < ApplicationRecord
+  validates_presence_of :name, :age
```

Comment:
Nice work validating these attributes! You are not required to create validations during an IC but it's good practice if you choose to include it!